scripts/generate_api_docs.py

```diff
@@ -19,100 +19,46 @@
-    "aea.helpers.pipe": "api/helpers/pipe.md",
-    "aea.helpers.profiling": "api/helpers/profiling.md",
-    "aea.helpers.sym_link": "api/helpers/sym_link.md",
-    "aea.helpers.temp_error_handler": "api/helpers/temp_error_handler.md",
-    "aea.helpers.win32": "api/helpers/win32.md",
-    "aea.helpers.yaml_utils": "api/helpers/yaml_utils.md",
-    "aea.identity.base": "api/identity/base.md",
-    "aea.mail.base": "api/mail/base.md",
-    "aea.protocols.base": "api/protocols/base.md",
-    "aea.protocols.dialogue.base": "api/helpers/dialogue/base.md",
-    "aea.protocols.generator.base": "api/protocols/generator/base.md",
-    "aea.protocols.generator.common": "api/protocols/generator/common.md",
-    "aea.protocols.generator.extract_specification": "api/protocols/generator/extract_specification.md",
-    "aea.protocols.generator.validate": "api/protocols/generator/validate.md",
-    "packages.fetchai.protocols.default.custom_types": "api/protocols/default/custom_types.md",
-    "packages.fetchai.protocols.default.dialogues": "api/protocols/default/dialogues.md",
-    "packages.fetchai.protocols.default.message": "api/protocols/default/message.md",
-    "packages.fetchai.protocols.default.serialization": "api/protocols/default/serialization.md",
-    "packages.fetchai.protocols.signing.custom_types": "api/protocols/signing/custom_types.md",
-    "packages.fetchai.protocols.signing.dialogues": "api/protocols/signing/dialogues.md",
-    "packages.fetchai.protocols.signing.message": "api/protocols/signing/message.md",
-    "packages.fetchai.protocols.signing.serialization": "api/protocols/signing/serialization.md",
-    "packages.fetchai.protocols.state_update.dialogues": "api/protocols/state_update/dialogues.md",
-    "packages.fetchai.protocols.state_update.message": "api/protocols/state_update/message.md",
-    "packages.fetchai.protocols.state_update.serialization": "api/protocols/state_update/serialization.md",
-    "aea.registries.base": "api/registries/base.md",
-    "aea.registries.filter": "api/registries/filter.md",
-    "aea.registries.resources": "api/registries/resources.md",
-    "aea.skills.base": "api/skills/base.md",
-    "aea.skills.behaviours": "api/skills/behaviours.md",
-    "aea.skills.tasks": "api/skills/tasks.md",
-    "packages.fetchai.skills.error.handlers": "api/skills/error/handlers.md",
-    "aea.test_tools.generic": "api/test_tools/generic.md",
-    "aea.test_tools.test_cases": "api/test_tools/test_cases.md",
-    "aea.test_tools.test_skill": "api/test_tools/test_skill.md",
+IGNORE_NAMES = {r"^__init__\.py$", r"^__version__\.py$", r"^py\.typed$", r"^.*_pb2.py$"}
+IGNORE_PREFIXES = {
+    Path("aea", "cli_gui"),
+    Path("aea", "cli"),
+    Path("aea", "connections", "scaffold"),
+    Path("aea", "contracts", "scaffold"),
+    Path("aea", "protocols", "scaffold"),
+    Path("aea", "skills", "scaffold"),
+    Path("aea", "decision_maker", "scaffold"),
```

Comment:
processing aea/decision_maker/scaffold.py... Running with dotted path=aea.decision_maker.scaffold and dest_file=docs/api/decision_maker/scaffold.md... Done!

still saw this...

Reply:
In test tools we should ignore click_testing